x/concentrated-liquidity/types/events.go

```diff
@@ -49,5 +49,9 @@ const (
 	AttributeKeyUptimeGrowthOppositeDirectionOfLastTraversal       = "uptime_growth"
 	AttributeNewOwner                                              = "new_owner"
 
+	IncentiveEmissionPlaceholderName   = "concentrated_liquidity_incentive_emission"
 	IncentiveTruncationPlaceholderName = "concentrated_liquidity_incentive_truncation"
+
+	SpreadRewardEmissionPlaceholderName   = "concentrated_liquidity_spread_reward_emission"
+	SpreadFactorTruncationPlaceholderName = "concentrated_liquidity_sptread_factor_truncation"
```

Comment:
@alessandrolomanto @niccoloraspa 

I added 3 new counters/logs that work similarly:
- `concentrated_liquidity_incentive_emission`
   * This is emitted when incentives accumulator is updated (works only for pools 1423 and 1213 that are at the border)
      * Let's please set up alert for pool 1423 ONLY
- `concentrated_liquidity_spread_reward_emission`
   * This is emitted when spread rewads accumulator is updated
      * No alerts needed at the time
- `concentrated_liquidity_sptread_factor_truncation`
    * This is emitted when spread rewards accumulator is truncated and is not updated
    * Let's set up an alert for ANY pool